addons/account/models/account_move.py

```diff
@@ -614,6 +614,14 @@ def _sequence_year_range_monthly_regex(self):
         tracking=True,
         help="It indicates that the invoice/payment has been sent or the PDF has been generated.",
     )
+    is_move_sent_string = fields.Selection(
+        selection=[
+            ("sent", "Sent"),
+            ("not_sent", "Not Sent")
+        ],
+        copy=False,
+        compute="_compute_is_move_sent_string"
+    )
```

Comment:
Rule of thumb when working with quotes in **python** --> Double quote for what the user will see, and single quote for other cases :smile: 
```suggestion
        selection=[
            ('sent', "Sent"),
            ('not_sent', "Not Sent")
        ],
        copy=False,
        compute='_compute_is_move_sent_string'
    )
```